docs/composer.rst

```diff
@@ -82,10 +82,10 @@ defining an extension as follows:
 
     class ExtendedDataDocument:
         """Data ES document extensions."""
-        source = dsl.String(index='not_analyzed')
-        species = dsl.String(index='not_analyzed')
-        build = dsl.String(index='not_analyzed')
-        feature_type = dsl.String(index='not_analyzed')
+        source = dsl.Keyword()
+        species = dsl.Text()
```

Comment:
Just curious: what is the difference between the `dsl.Text` in `dsl.Keyword`?

Reply:
https://www.elastic.co/blog/strings-are-dead-long-live-strings